includes/icon-blocks.html

```diff
@@ -46,16 +46,19 @@
               alt=iconBlocks.alt
             %}
           </figure>
-          {%
-            include headline.html
-            text=item.headline
-            level=iconBlocksLevel
-            classes=iconBlocksClasses
-          %}
-          <p>{{ item.copy }}</p>
-        </div>
-        <!-- End Shared Icon Blocks -->
+            {%
+              include headline.html
+              text=item.headline
+              level=iconBlocksLevel
+              classes=iconBlocksClasses
+            %}
+
+          <div class="icon-blocks__bulletpoints">
```

Comment:
Finde das Naming hier eher falsch. Ist ja nach wie vor die copy, in der vielleicht auch Bulletpoints vorkommen.